### PR TITLE
EZEE-2853: Fixed using preferedded date in scheduler

### PR DESCRIPTION
**Pull request by GrabowskiM**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZEE-2853
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Added using preferred date format in scheduler.
Also: https://github.com/ezsystems/ezplatform-page-builder/pull/469

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/js/scripts/admin.picker.js [hunk 1]
```diff
@@ -4,10 +4,11 @@
     const SELECTOR_FORM_INPUT = '.ez-picker__form-input';
     const SELECTOR_CLEAR_BTN = '.ez-picker__btn--clear-input';
     const pickers = [...doc.querySelectorAll(SELECTOR_PICKER)];
+    const { convertDateToTimezone, formatShortDateTime, userPreferedTimezone } = eZ.helpers.timezone;
     const pickerConfig = {
         enableTime: true,
         time_24hr: true,
-        formatDate: (date) => new Date(date).toLocaleString(),
+        formatDate: (date) => formatShortDateTime(convertDateToTimezone(new Date(date), userPreferedTimezone, true)),
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/admin.picker.js]: The same comments like in the related PR

File: src/bundle/Resources/public/js/scripts/admin.picker.js [hunk 2]
```diff
@@ -4,10 +4,11 @@
     const SELECTOR_FORM_INPUT = '.ez-picker__form-input';
     const SELECTOR_CLEAR_BTN = '.ez-picker__btn--clear-input';
     const pickers = [...doc.querySelectorAll(SELECTOR_PICKER)];
+    const { formatShortDateTime } = eZ.helpers.timezone;
     const pickerConfig = {
         enableTime: true,
         time_24hr: true,
-        formatDate: (date) => new Date(date).toLocaleString(),
+        formatDate: (date) => formatShortDateTime(date, null),
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/admin.picker.js]: `null` can be skipped

Comment (GrabowskiM) [reply src/bundle/Resources/public/js/scripts/admin.picker.js]: It's needed here

**Comment by lserwatka** [comment]
Could you merge it up?

**Pull request merged**